### ISSUE TITLE
utterly confused?

### ISSUE THREAD

**Issue by skyisthela** [mat1jaczyyy/Keyboard-Inspector#27]
i have an rk61, i decided to try this software out on a whim, when i did a test the bottom left graph peaked at 50 something hz while the middle is consistently 1000hz and the right one is arbitrary, sometimes a 1000 sometimes 1300 or 1900 or whatever it's not consistent. i did another test only using NUMBERS and the left graph showed 990-999 hz consistently, while the middle and right showed 1000hz. what is going on lol

**Comment by mat1jaczyyy** [comment]
This issues forum is meant for development of the software. For general usage questions the [Discord](https://discord.gg/kX4cJQH5Zn) is better suited.

That said, how the graphs are generated and what they mean in the input data is explained in technical detail [here](https://github.com/mat1jaczyyy/Keyboard-Inspector/blob/master/DOCUMENTATION.md#time-domain). The general takeaway though is that you probably need some fundamental understanding of signal analysis to be able to interpret these accurately. The [README](https://github.com/mat1jaczyyy/Keyboard-Inspector#getting-started) does talk about these in simpler terms, and I do think we should provide some more examples rather than theory talk in general.

It's hard to help you with your results directly if you don't share a .kbi file or at the very least a screenshot of the analysis.

**Comment by skyisthela** [comment]
> This issues forum is meant for development of the software. For general usage questions the [Discord](https://discord.gg/kX4cJQH5Zn) is better suited.
> 
> That said, how the graphs are generated and what they mean in the input data is explained in technical detail [here](https://github.com/mat1jaczyyy/Keyboard-Inspector/blob/master/DOCUMENTATION.md#time-domain). The general takeaway though is that you probably need some fundamental understanding of signal analysis to be able to interpret these accurately. The [README](https://github.com/mat1jaczyyy/Keyboard-Inspector#getting-started) does talk about these in simpler terms, and I do think we should provide some more examples rather than theory talk in general.
> 
> It's hard to help you with your results directly if you don't share a .kbi file or at the very least a screenshot of the analysis.

yeah mb man, also for the record i did read the readme, i still understood nothing lol
here's a screenshot of just randomly pressing letters on my board
![image](https://github.com/mat1jaczyyy/Keyboard-Inspector/assets/150245867/3e286cab-c9c2-43b7-a217-fe1862fa1b29)
here's a screenshot of using numbers 
![image](https://github.com/mat1jaczyyy/Keyboard-Inspector/assets/150245867/8780d75e-bbc2-4024-b2e7-28e83144ece2)

note that, it was still incredibly arbitrary results are different nearly all the time

**Comment by mat1jaczyyy** [comment]
If you get wildly different results every time, send a couple of different files. It's easier if you save the file from the File menu as I can then load it and play around with the analysis parameters.

Judging off this screenshot, looks like you have decent 1000Hz, but extremely severe chord splitting issues.
